tikv-configuration-file.md

```diff
@@ -996,6 +996,23 @@ Configuration items related to `rocksdb.defaultcf`
 + Default value: `"256GB"`
 + Unit: KB|MB|GB
 
+### `enable-compaction-guard`
+
++ Enable compaction guard, which is an optimization to split SST files at TiKV region boundaries. The optimization can help reduce compaction IO, and allow us to use larger SST file size (thus less SST files overall) while making sure we can still efficiently cleanup stale data on region migration.
```

Comment:
* "compaction guard" is the proper noun of the optimization. maybe not to use "the"?
* "region" is a concept in TiKV, something like a "slice" or "shard". maybe not to capitalize?
* usage of "you" sound weird. let's use "TiKV" instead for each of its occurrence?

Reply:
need to apply the same change below.

Reply:
+ The standard spelling for the concept of "Region" in TiKV is "Region", with the first letter capitalized. See the classical blog post [TiDB Internal (III) - Scheduling](https://en.pingcap.com/blog/2017-07-20-tidbinternal3) or other TiKV docs for details.
+ In technical documentation, we should try to avoid the first person (I/We) and use "you" instead (when applicable).  See [Google developer documentation style guide](https://developers.google.com/style/person). In this case, if "us" refers to TiKV, let's use TiKV.